### PR TITLE
`README.riscos`: Improve the document

### PR DESCRIPTION
**Pull request by rwp0**
Enhance formatting, minor clarifications etc

https://perldoc.perl.org/perlriscos

### CODE REVIEW

**Comment by rwp0** [comment]
[Cygwin](https://github.com/Perl/perl5/actions/runs/8508198145/job/23301685544?pr=22113) fails for an unrelated reason:
> 0 [main] perl 2982 child_info_fork::abort: address space needed by 'mro.dll' (0x400000) is already occupied

**Comment by tonycoz** [comment]
> [Cygwin](https://github.com/Perl/perl5/actions/runs/8508198145/job/23301685544?pr=22113) fails for an unrelated reason:
> 
> > 0 [main] perl 2982 child_info_fork::abort: address space needed by 'mro.dll' (0x400000) is already occupied

Try rebasing on blead now.

File: README.riscos [hunk 1]
```diff
@@ -11,19 +11,17 @@ perlriscos - Perl version 5 for RISC OS
...
+complicated by the need to cross-compile. There is a binary version of
+perl available from L<https://www.cp15.org/perl/> which you may wish to
 use instead of trying to compile it yourself.
 
 =head1 BUILD
 
-You need an installed and working gccsdk cross compiler
-L<http://gccsdk.riscos.info/> and REXEN
-L<http://www.cp15.org/programming/>
+You need an installed and working L<GCCSDK|https://gccsdk.riscos.info/>
```
Comment (tonycoz) [inline README.riscos]: The `https://gccsdk.riscos.info/` URL redirects to a 404.

I suspect http://www.riscos.info/downloads/gccsdk/ is the best current option.

**Comment by tonycoz** [comment]
I'll be greatly surprised if riscos builds still worked, though there are modern risc os systems -- raspberry pi can run [risc os open](https://www.riscosopen.org/content/downloads/raspberry-pi)

File: README.riscos (hunk 1 above)
Comment (rwp0) [reply README.riscos]: Yes, will amend to https://www.riscos.info/downloads/gccsdk/latest/ then

**Comment by rwp0** [comment]
@tonycoz I replaced the link and rebased my branch.

Could you please review again and approve if everything is alright?

Thanks

**Review by tonycoz — APPROVED** [review]

**Pull request merged**